other_types_of_ownership/p2wpkh_pay_to_witness_public_key_hash.md

```diff
@@ -1,20 +1,20 @@
 ## P2WPKH (Pay to Witness Public Key Hash) {#p2wpkh-pay-to-witness-public-key-hash}
 
-In 2015, Pieter Wuille introduced a new feature to bitcoin called **Segregated Witness** or shorter, **segwit**. Basically, Segregated Witness move the proof of ownership from the **scriptSig** part of the transaction, to a new part called the **witness** of the input.
+In 2015, Pieter Wuille introduced a new feature to bitcoin called **Segregated Witness**, also known by it's abbreviated name, **Segwit**. Basically, Segregated Witness moves the proof of ownership from the **scriptSig** part of the transaction to a new part called the **witness** of the input.
 
-There is several reason to use this new scheme, here is a summary, for more details go to [https://bitcoincore.org/en/2016/01/26/segwit-benefits/](https://bitcoincore.org/en/2016/01/26/segwit-benefits/)
+There are several reasons why it is beneficial to use this new scheme, a summary of which are presented below. For more details visit [https://bitcoincore.org/en/2016/01/26/segwit-benefits/](https://bitcoincore.org/en/2016/01/26/segwit-benefits/).
 
-*   **Third party Malleability Fix:** Previously, a third party could change the transaction id of your transaction before it was confirmed. Not anymore.
-*   **Linear sig hash scaling:** Signing a transaction would require hashing the whole transaction for every input. This was a potential DDoS vector attack for large transactions.
-*   **Signing of input values:** The amount that is spent in an input is also signed, meaning the signer can’t be tricked about the amount of fee that is actually paid.
-*   **Capacity increase:** It will now be possible to have more than 1MB of transactions every 10 min, up to around 1.75.
-*   **Fraud proof:** Will be developed later, but SPV wallet will be able to validate more consensus rule than just following the longest chain.
+*   **Third party Malleability Fix:** Previously, a third party could change the transaction id of your transaction before it was confirmed. This can not occur under Swgwit.
+*   **Linear sig hash scaling:** Signing a transaction used to require hashing the whole transaction for every input. This was a potential DDoS vector attack for large transactions.
+*   **Signing of input values:** The amount that is spent in an input is also signed, meaning that the signer can’t be tricked about the amount of fees that are actually being paid.
+*   **Capacity increase:** It will now be possible to have more than 1MB of transactions in each block (which are created every 10 minutes on average). Segwit increases this capacity by a factor of about 1.7, although this depends upon the types of transaction included in each block. More information can be found at [https://segwit.org/is-segwit-a-block-size-increase-705df6a8731d](https://segwit.org/is-segwit-a-block-size-increase-705df6a8731d).
```

Comment:
> The actual size of a block under Segwit depends on the kind of transactions being included, however the figure of 1.7MB was based on the average transaction profile in January 2015. At the time of writing (Nov 2016), it would be around 2.1MB. (https://www.weusecoins.com/eli-segwit/)